src/components/ContactForm/index.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from "react";
+import Typography from "root/components/Typography";
+import Button from "root/components/Button";
+import classNames from "classnames";
+
+import "./index.css";
+
+const ContactForm = () => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const [enableSubmit, setEnableSubmit] = useState(false);
+  const [enableErrors, setEnableErrors] = useState(false);
+
+  const validEmail = value => {
+    return value && /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(value);
+  };
+
+  const onChangeName = event => {
```

Comment:
Consider this. instead of creating a useState for each field and subsequent handlers, we can use this hook.

```
import { useState } from "react";

export default function useForm() {
  const [formValues, setFormValues] = useState({});

  const handleChange = event => {
    event.persist();

    setFormValues({ ...formValues, [event.target.name]: event.target.value });
  };

  return {
    handleChange,
    formValues
  };
}
```

Reply:
Thanks! I've just updated.